i18n/zh/docusaurus-plugin-content-docs/version-v1.1/test-report/pool-coordinator-test.md

```diff
@@ -0,0 +1,168 @@
+---
```

Comment:
please add to `i18n/zh/docusaurus-plugin-content-docs/current/test-report/` dir instead of `version-v1.1`